pub_weight/src/main.rs

```diff
@@ -0,0 +1,62 @@
+use bincode;
+use failure::Error;
+use futures::stream::Stream;
+use mqtt::{control::variable_header::ConnectReturnCode, packet::*, Decodable, Encodable};
+use tokio::{runtime, timer::Interval};
+use uuid::Uuid;
+
+use std::{io::Write, net::TcpStream, time::Duration};
+
+use pirust_common::Weight;
+
+fn generate_client_id() -> String {
+    format!("/MQTT/rust/{}", Uuid::new_v4())
+}
+
+fn main() -> Result<(), Error> {
+    let server_addr = "192.168.1.44:1883";
```

Comment:
Could we make this variable configurable using for example program arguments: `--mqtt-server` ?

Reply:
Sure! Just left it as-is for the sake of simplicity, for now..
Feel free to refactor it with [clap-rs](https://github.com/clap-rs/clap)

Reply:
I'm willing definitely to implement this :)